content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -39,13 +40,40 @@ Open a shell to the Node in your cluster. How you open a shell depends on how
 you set up your cluster. For example, if you are using Minikube, you can open a
 shell to your Node by entering `minikube ssh`.
 
-In your shell, create a `/mnt/data` directory:
+In your shell on that Node, create a `/mnt/data` directory:
+
+```shell
+# This assumes that your Node uses "sudo" to run commands
+# as the superuser
+sudo mkdir /mnt/data
```

Comment:
git rid of the explanation in the command

Reply:
@stewart-yu can you explain why you're not keen?